test/fixtures/users.yml

```diff
@@ -379,3 +379,4 @@ state1_api_proxy:
   api_enabled: true
   role: 'public_health_enroller'
   is_api_proxy: true
+  
```

Comment:
I think there might be a space or something on this line

Reply:
fixed in https://github.com/SaraAlert/SaraAlert/pull/805/commits/bbcf27d175a0137087af74da9d1cdd635f421c09